controllers/array_action/array_mediator_svc.py

```diff
@@ -65,6 +65,10 @@
 ENDPOINT_TYPE_MASTER = 'master'
 ENDPOINT_TYPE_AUX = 'aux'
 
+ENDPOINT_TYPE_PRODUCTION = 'production'
```

Comment:
how about UT?
(relevant for all file)

Reply:
Added unit test

Reply:
I added UT for get and create replication functionality. @matancarmeli7 suggested to open separate task in order to cover delete and promote replication.

Reply:
I don't think it's a good idea to separate implementation and UT, its a recipe for bugs in the code

Reply:
UT for mediator are ready.